bin/tests/bootstrap.bats

```diff
@@ -55,4 +55,27 @@ teardown () {
     bin/bootstrap.sh 
     run bin/bootstrap.sh
     assert_success
-}
\ No newline at end of file
+}
+
+@test "executing bootstrap.sh creates a valid loose schema" {
+    git init "$ENV_DIR"
+    run bin/bootstrap.sh
+    assert_success
+    assert_file_exist "$ENV_DIR/.vscode/values-schema.yaml"
+    assert_file_exist "$PWD/values-schema.yaml"
+
+    result=$(yq r "$PWD/values-schema.yaml" '**.required.' | wc -l)
+    [ "$result" -ne 0 ]
+    result=$(yq r "$PWD/values-schema.yaml" 'properties.toolsVersion' | wc -l)
+    [ "$result" -ne 0 ]
+    result=$(yq r "$PWD/values-schema.yaml" 'properties.cluster' | wc -l)
+    [ "$result" -ne 0 ]
+
+    result=$(yq r "$ENV_DIR/.vscode/values-schema.yaml" '**.required.' | wc -l)
+    [ "$result" -eq 0 ]
+    result=$(yq r "$ENV_DIR/.vscode/values-schema.yaml" 'properties.toolsVersion' | wc -l)
+    [ "$result" -eq 0 ]
+    result=$(yq r "$ENV_DIR/.vscode/values-schema.yaml" 'properties.cluster' | wc -l)
+    [ "$result" -eq 0 ]
```

Comment:
Why are we suddenly addings tests that only test the workings of yq here? Can we discuss these things first? This is undesirable

Reply:
@Morriz was discussed as desirable during standup. 
It is not testing the workings of yq, but verifying that the original yaml contains these keys, and the new yaml does not contain these keys.
So it verifies that the (now speed optimized) removes these fields correctly.

Reply:
It is testing nothing of our code. It is merely testing if yq does what you ask it to do. Call me if you want further clarification.

Reply:
@Morriz we could've discussed this during stand-up. It can't be that you have to micro-manage every piece of code, it's like you don't trust anyone.

I'm just going to lay out how I experienced it. First you create an issue that the performance of the bats testing is not good. That's true, it takes about 30 seconds. Of course there's yq to blame, but it's not nice to reproach me about code you added. Then when I mention the performance gain of piping to jq, you don't believe me.

Now this test will actually test if the behaviour is the same of pre-jq and post-jq. It's about 6 lines of code and to me it seems like this would improve coverage because the logic is actually in the function `generate_loose_schema`. If someone adds a different piece of logic, say, we don't need to remove the required keys anymore (I don't actually understand why those need to be removed in the first place, since it's valid json schema), then this test will complain and that makes it easier to debug. 

Of course, if there is anything wrong about my reasoning then I'm open to learn from you, as always.